File: MakeAWish/app/src/main/java/tt/co/jesses/makeawish/NotificationActivity.java
package tt.co.jesses.makeawish;

import com.google.firebase.analytics.FirebaseAnalytics;

import android.app.Activity;
import android.os.Bundle;

/**
 * Created by jessescott on 2017-02-22.
 */

public class NotificationActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_notification);

        FirebaseAnalytics.getInstance(this.getApplicationContext()).setCurrentScreen(NotificationActivity.this, "NotificationActivity", NotificationActivity.class.getSimpleName());
    }

}

File: MakeAWish/app/src/main/java/tt/co/jesses/makeawish/NotificationActivity.kt
package tt.co.jesses.makeawish

import android.app.Activity
import android.os.Bundle
import com.google.firebase.analytics.FirebaseAnalytics

/**
 * Created by jessescott on 2017-02-22.
 */

class NotificationActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_notification)

        FirebaseAnalytics.getInstance(this.applicationContext).setCurrentScreen(this@NotificationActivity, "NotificationActivity", NotificationActivity::class.java.simpleName)
    }

}
